### PR TITLE
Store time data in UTC

### LINKED ISSUE #1602: DateTimeTzFields are serialized with incorrect times
Example saved this time in a `ChannelMetaDataCache` object (at around 14:00 local time)

`2017-06-06 14:44:12.582879(America/Los_Angeles)`

But it gets serialized as

`2017-06-06T07:44:12.582-07:00`

Expected is

`2017-06-06T14:44:12.582-07:00`

### PR DESCRIPTION
**Pull request by rtibbles**
## Summary

Ensure that time data is stored in UTC, not local time, to ensure sortability, and proper reinstantiation.

## TODO

- [X] Have tests been written for the new code?

## Issues addressed

Fixes #1602

### CODE REVIEW

File: kolibri/core/serializers.py [hunk 1]
```diff
@@ -7,6 +9,8 @@ class DateTimeTzField(DateTimeField):
     def to_internal_value(self, data):
         data = super(DateTimeTzField, self).to_internal_value(data)
         tz = timezone.get_current_timezone()
+        if not data.tzinfo:
+            data = timezone.make_aware(data, pytz.utc)
```
Comment (benjaoming) [inline kolibri/core/serializers.py:13]: This is missed by the code coverage, perhaps add a test of something that doesn't have tz info?

**Comment by benjaoming** [comment]
Comment regarding test coverage, not a blocker in case it's vital to get this merged.

File: kolibri/core/fields.py [hunk 2]
```diff
@@ -25,7 +25,8 @@ def create_timezonestamp(value):
         tz = value.tzinfo.zone
     else:
         tz = timezone.get_current_timezone().zone
-    date_time_string = value.strftime(date_time_format)
+        value = timezone.make_aware(value, timezone.get_current_timezone())
+    date_time_string = value.astimezone(pytz.utc).strftime(date_time_format)
```
Comment (benjaoming) [inline kolibri/core/fields.py:29]: Isn't this part what actually fixes the issue?

Comment (rtibbles) [reply kolibri/core/fields.py:29]: Yes - the other change was due to the fact that Travis was running its tests with a naive datetime for some reason (although it's possible that it doesn't always do that) - can add the suggested test!

**Pull request merged**